_el/exercises.md

```diff
@@ -336,172 +204,52 @@ piece of code with the output it produces".
-
-[[Steg2016b](#CITE)] is a good starting point for a code style rubric,
-while [[Luxt2009](#CITE)] looks at peer review in programming classes
-more generally. If you are going to have students do reviews, use
-calibrated peer review ([s:individual-peer](#SECTION)) so that they
-have models of what good feedback should look like.
-
-> **Code Review**
-> 
-> Using the rubric provided, mark each line of the code below.
-> 
+Οι ασκήσεις *Αντιστοίχισης Πολλά-προς-πολλά* είναι παρόμοιες, αλλά οι λίστες δεν είναι ιδίου μήκους και επομένως ορισμένα στοιχεία μπορεί να ταιριάζουν με πολλά άλλα, ενώ άλλα μπορεί να μην ταιριάζουν σε κάποιο. Και τα δύο είδη απαιτούν από τους μαθητές να χρησιμοποιούν συλλογιστική σκέψη υψηλότερης τάξης, αλλά οι ασκήσεις αντιστοίχισης πολλά-προς-πολλά είναι πιο δύσκολες επειδή οι μαθητές δεν μπορούν να κάνουν πρώτα τις εύκολες αντιστοιχίσεις ώστε να μειωθεί ο συνολικός χώρος αναζήτησης (δηλαδή, υπάρχει ένα υψηλότερο γνωστικό φορτίο).
+
+Τα προβλήματα αντιστοίχισης μπορούν να υλοποιηθούν με την υποβολή από μαθητές καταλόγων με ζευγάρια που εκφράζουν τις αντιστοιχίσεις ως κείμενο (όπως για παράδειγμα "A3, B1, C2"), αλλά αυτός είναι ένας γενικά αδέξιος τρόπος που είναι επιρρεπής σε λάθη. Ζητώντας να αναγνωρίσουν ένα σύνολο σωστών ζευγών υπό τη μορφή ΕΠΕ είναι ακόμα χειρότερο, καθώς είναι δυστυχώς πολύ εύκολο να το διαβάζει κανείς εσφαλμένα.
+
+Η *Κατάταξη* είναι μια ειδική περίπτωση που (ελαφρώς) ταιριάζει σε απάντηση μέσω λιστών, καθώς το μυαλό μας είναι πολύ καλά εκπαιδευμένο στο να εντοπίζει σφάλματα ή ανωμαλίες σε ακολουθίες. Δώστε στο μαθητή πολλά στοιχεία και ζητήσετε να τα κατατάξουν από τα ταχύτερα στα πιο αργά, από τα πιο ανθεκτικά στα πιο εύθραυστα, και ούτω καθεξής. Η πρώτη μορφή ενδείκνυται περισσότερο για την ανάκληση πληροφορίας (π.χ. την αναγνώριση των ονομάτων διαφόρων αλγορίθμων κατάταξης και γνωρίζοντας τις ιδιότητές τους), ενώ η δεύτερη τείνει περισσότερο προς τη συλλογιστική και την κρίση.
+
+Η *Σύνοψη* απαιτεί επίσης από τους εκπαιδευόμενους να χρησιμοποιούν συλλογιστική υψηλότερης τάξης, και τους δίνει την ευκαιρία να εξασκήσουν μια δεξιότητα που είναι πολύ χρήσιμη όταν στοχεύει περισσότερη στην ανίχνευση σφαλμάτων παρά στη διόρθωσή τους. Για παράδειγμα, η ερώτηση προς τους μαθητές μπορεί να είναι: "Ποια πρόταση περιγράφει καλύτερα τον τρόπο με τον οποίο η έξοδος της f αλλάζει όταν το x παίρνει τιμές μεταξύ 0 και 10;" και δίνοντάς τους μετά πολλές επιλογές ως ερώτηση πολλαπλών επιλογών.
+
+Μπορείτε επίσης να ζητήσετε πολύ σύντομες απαντήσεις ελεύθερου κειμένου σε ερωτήσεις που έχουν πολύ στενό αντικείμενο, όπως για παράδειγμα "Ποιο είναι το βασικό χαρακτηριστικό ενός ευσταθούς αλγορίθμου ταξινόμησης;" Δεν μπορούμε ακόμα να αυτοματοποιήσουμε πλήρως τη βαθμολόγηση τέτοιων ερωτήσεων χωρίς ένα
+απογοητευτικό αριθμό ψευδών θετικών (αποδοχή λανθασμένων απαντήσεων) και ψευδών αρνητικών (απορρίπτοντας τα σωστά), αλλά ενδείκνυνται για αυτοαξιολόγηση ([s:individual-peer](#SECTION)).
```

Comment:
απορρίπτοντας τα σωστά --> απόρριψη των σωστών
αυτοαξιολόγηση--> αξιολόγηση μεταξύ συμμαθητών 
peer grading:(http://www.greek-language.gr/certification/research/lexicon/show.html?id=149)

Reply:
Τhanks @pdelias ! Fixed in a new commit.